app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module V1
+    class TargetsController < ApplicationController
+      before_action :authenticate_user!
+
+      def index
+        @targets = Target.where(user_id: current_user.id)
+      end
+
+      def create
+        @target = Target.create!(target_params.merge(user_id: current_user.id))
```

Comment:
Similar as the previous comment, you can also do:
`current_user.targets.create!(target_params)`